linkml_model/model/schema/meta.yaml

```diff
@@ -1333,6 +1360,15 @@ classes:
     close_mappings:
       - qb:HierarchicalCodeList
 
+  alias_definition:
+    description: >-
+      object that contains meta data about a synonym or alias including where it came from (source)
+      and its scope (narrow, broad, etc.)
+    slots:
+      - alias_name
+      - alias_source
+      - alias_scope
```

Comment:
scope: This naming convention is very obo-format esque and not really used elsewhere. I would favor alias_type (perhaps alias_property/predicate/category)

Reply:
I removed the 'alias_' convention and this turned into 'type' (and changed 'alias_scope_enum' to 'type_enum').